.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ env:
 
 jobs:
   lint:
-    runs-on: ubuntu-16.04
```

Comment:
Use reverted version

Reply:
This ubuntu version is deprecated in GitHub actions...

Reply:
So we should have a ticket to use docker runner. I will create one

Reply:
#16 16